docs/source/usage/examples.rst

```diff
@@ -18,6 +18,7 @@ C++
 - `3a_write_thetaMode_serial.cpp <https://github.com/openPMD/openPMD-api/blob/dev/examples/3a_write_thetaMode_serial.cpp>`_: write an azimuthally decomposed mesh
 - `4_read_parallel.cpp <https://github.com/openPMD/openPMD-api/blob/dev/examples/4_read_parallel.cpp>`_: MPI-parallel mesh read
 - `5_write_parallel.cpp <https://github.com/openPMD/openPMD-api/blob/dev/examples/5_write_parallel.cpp>`_: MPI-parallel mesh write
+- `5a_write_parallel.cpp <https://github.com/openPMD/openPMD-api/blob/dev/examples/5a_write_parallel.cpp>`_: creates 1D or 2D arrays, with each rank having a few blocks to write to
```

Comment:
Shall we maybe name this `8_benchmark_write1D2D_parallel`?

Reply:
OK